src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -34,21 +54,65 @@ class ApiProjectKey:
     dsn_public: str = ""
 
 
+@dataclass
+class ApiTeam:
+    id: int = -1
+    status: ApiTeamStatus = ApiTeamStatus.VISIBLE
+    organization_id: int = -1
+    slug: str = ""
+
+
+@dataclass
+class ApiTeamMember:
+    id: int = -1
+    is_active: bool = False
+    role: TeamRole = field(default_factory=TeamRole)
+    project_ids: List[int] = field(default_factory=list)
+    scopes: List[str] = field(default_factory=list)
+    team: ApiTeam = field(default_factory=ApiTeam)
+
+
+@dataclass
+class ApiProject:
+    id: int = -1
+    slug: str = ""
+    name: str = ""
+    organization_id: int = -1
+    status: ApiProjectStatus = ApiProjectStatus.VISIBLE
+
+
 @dataclass
 class ApiOrganizationMember:
+    id: int = -1
+    organization_id: int = -1
     # This can be null when the user is deleted.
-    user_id: Optional[int]
-    pass
+    user_id: Optional[int] = None
+    teams: List[ApiTeamMember] = field(default_factory=list)
+    role: str = ""
+    projects: List[ApiProject] = field(default_factory=list)
+    scopes: List[str] = field(default_factory=list)
 
 
 @dataclass
 class ApiOrganization:
     slug: str = ""
     id: int = -1
-    # exists iff the organization was queried with a user_id context, and that user_id
+    # exists if and only if the organization was queried with a user_id context, and that user_id
     # was confirmed to be a member.
     member: Optional[ApiOrganizationMember] = None
 
+    # Represents the full set of teams and proejcts associated with the org.
+    teams: List[ApiTeam] = field(default_factory=list)
+    projects: List[ApiProject] = field(default_factory=list)
+
+    allow_joinleave: bool = False
```

Comment:
Unpacking the `flags` field of organizations.  It didn't seem necessary to nest them.

Reply:
I agree that unpacking them is a good choice. We also unpack them in API responses.